Bottom: ca9f243a34950c1a75ece3674cf8d5867494a994
Top:    363453232d5e45da67b20c2d74fc207df97266df
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-08-14 16:24:52 -0500

Enable file-based secrets provider


---
diff --git a/docker/buildbot-master/buildbot/master.cfg b/docker/buildbot-master/buildbot/master.cfg
index f9845106..8f045992 100644
--- a/docker/buildbot-master/buildbot/master.cfg
+++ b/docker/buildbot-master/buildbot/master.cfg
@@ -5,7 +5,7 @@
 # 'master.cfg' in your buildmaster's base directory.
 
 import os
-from buildbot.plugins import steps,util
+from buildbot.plugins import secrets,steps,util
 
 os.environ['LONG_RUN_TIMEOUT']='14400'
 
@@ -22,6 +22,9 @@ c['change_source'] = []
 # --master option)
 c['protocols'] = {"pb": {"port": int(os.getenv('BUILDBOT_WORKER_PORT',9989))}}
 
+### SECRETS MANAGEMENT
+c['secretsProviders'] = [secrets.SecretInAFile(dirname="/run/secrets")]
+
 ####### STATUS TARGETS
 
 # 'status' is a list of Status Targets. The results of each build will be
